ClinGen/clincoded
# Connects to #1343. Connects to #1753. Connects to #1822. Connects to #1830.

**Steps to test #1343:**
1. Start a new interpretation in the VCI (example variant: 17000).
2. Go to the **Population** tab and expect to see individual explanation boxes for the **BA1, BS1 and PM2** criteria.
3. In each one of those explanation boxes, enter different sample information. Save and expect to see the different explanations to be stored individually.
4. Go to **View Summary** and expect to see those individual explanations being shown next to their respective criteria.

**Steps to test #1753 and #1822:**
1. Return to the Interpretation view and again select the **Population** tab.
2. At the bottom of the tab, click on the **Add PMID** button and enter a PMID (e.g. 1234).
3. Save to exit the PMID modal and expect to see a **Criteria** selection dropdown with a list of criteria codes pertinent to the evaluation category (_Population Criteria Evaluation_ in this case).
4. Select a code and enter some information in the **Evidence** text box.
5. Save and expect to see the criteria code pertinent to the evidence being stored successfully.

**Steps to test #1830:**
1. Sign into the curation interfaces and assign the **10028** affiliation to yourself.
2. Sign out and sign in again.
3. Expect to see the affiliation selection dropdown with the **Hemostasis/Thrombosis EP** option being spelled correctly (instead of **Hemostatis/Thrombosis EP**) in the modal.
